src/Common/src/Interop/OSX/Interop.CoreServices.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+using CFStringRef = System.IntPtr;
+using CFArrayRef = System.IntPtr;
+using FSEventStreamRef = System.IntPtr;
+using size_t = System.IntPtr;
+using FSEventStreamEventId = System.UInt64;
+using CFTimeInterval = System.Double;
+using CFRunLoopRef = System.IntPtr;
+
+internal static partial class Interop
+{
+    internal static partial class CoreServices
+    {
+        /// <summary>
+        /// This constant specifies that we don't want historical file system events, only new ones
+        /// </summary>
+        internal const ulong kFSEventStreamEventIdSinceNow = ulong.MaxValue;
+
+        /// <summary>
+        /// This is the default system path for the CoreServices library that PInvoke will use
+        /// </summary>
+        private const string CoreServicesLibrary = "/System/Library/Frameworks/CoreServices.framework/CoreServices";
```

Comment:
Ditto on moving this to Interop.Libraries.cs.  Same goes for any other such library constants declared.